README.md

```diff
@@ -10,15 +10,43 @@ The purpose of this repository is to allow easy management of all Pop!\_OS relat
 
 Binary packages are hosted [on Launchpad](https://launchpad.net/~system76/+archive/ubuntu/pop/+packages). Many packages have source on Github, under the [Pop!\_OS organization](https://github.com/pop-os).
 
+## Developer Resources
+For instructions on how to build the shell:
+	* [Cosmic, our current gnome-based shell](https://github.com/pop-os/cosmic). 
+	* [Cosmic-Epoch, the new Rust based shell](https://github.com/pop-os/cosmic-epoch).
+
 To view a list of all included repositories, see [REPOS.md](./REPOS.md)
+Developer chat: https://chat.pop-os.org/
+
+## How to Make a Change
+First of all, thank you for taking interest in contributing to our community! Every bit helps, even if it's just contributing to our documentation. 
+
+### Find the Correct Repo
+Before you make a change, you need to find the relevant repository to make a contribution in. See the 'Developer Resources' section for help finding the correct one.  When in doubt, you can make an issue in this catch all repo. 
+
+### Make An Issue
+All changes should start with an issue so the change can be properly discussed. The more complex the issue, the more discussion would be required. If an issue doesn't already exist, here would be the place to start.
+
+### Make a Pull Request.
+Fork the repository, make your changes and then make a Pull Request! Don't forget to use detail and explain what your change does. 
+
+Every PR to Pop!_OS components requires approval from the engineering team (for code quality and architectural fit) and quality assurance team (for stability and UX sanity.) Anything that significantly impacts the user experience (e.g. new GUI features) may also require approval from the user experience team. 
+
+### Enjoy Your Contribution!
+The Pop!_OS CI server automatically builds the master/main branch of every git repository (every 15 minutes), and all packages from those git branches are published in the master staging apt repository. Packages are then released from master staging as regular updates via PRs to the repo-release repository, which contains a list of the current version of every package in the release repository: https://github.com/pop-os/repo-release (There is another CI job that checks this list and copies the listed versions of each package from the master staging repo to the release repo.)
```

Comment:
I see that you just copied and pasted a few paragraphs of what I wrote [in your issue](https://github.com/pop-os/website/issues/69) without any context. This information is regarding how our CI system works, and isn't necessarily relevant in this section of the guide, at least under a header that sounds like it's the end of the process.

Reply:
This is information on how long it takes to enjoy the fruits of their labor. Also, once their PR is merged, have they not reached the end of the process?

Reply:
I think the real issue is that "enjoying the fruits of their labor" is outside the scope of this document (although my point was also that they cannot "enjoy the fruits of their labor" on a fresh installation until _after_ the CI process is complete.)

I have fixed this by changing the header in https://github.com/pop-os/pop/pull/2659/commits/2a2505c4945cea048e69f99b17293f99eb9fbc8e.